app/presenters/content_item_presenter.rb

```diff
@@ -8,13 +8,20 @@ def initialize(content_item)
     @format = content_item["format"]
     @locale = content_item["locale"] || "en"
     @phase = content_item["phase"]
-    @links = content_item["links"] || {}
   end
 
   def available_translations
     sorted_locales(links["available_translations"])
   end
 
+  def links
```

Comment:
No, I just moved them from the [guide presenter](https://github.com/alphagov/service-manual-frontend/pull/92/files#diff-fe302837b4e13df9599084e6ef3366e3) to make them available to the toolkit presenter.